homeassistant/components/wiffi/translations/en.json

```diff
@@ -1,6 +1,7 @@
 {
     "config": {
         "abort": {
+            "already_configured": "Server port is already configured.",
```

Comment:
This should be added to the `strings.json` and next run `python -m script.translations develop`